### ISSUE TITLE
Components deleted reapears on project

### ISSUE THREAD

**Issue by htsnet** [FlutterFlow/flutterflow-issues#2653]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I am working on a project named SV-Demo. Yesterday I deleted (at least) 2 files from the project: CntMonth and PickerMonthYear components.
Today, I returned to the project and the components are present again.
I don´t know about other files. but these returned to the project.

### Expected Behavior

Files deleted doesn´t reapear.

### Steps to Reproduce

I don´t know how to reproduce.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFfhefq0It2sdtE1oSIK/palmMXOU8cTL4zlO1HcxAYGO/yPbJ+PsnBSBBWTe29TVNhHFL+/TsdpNbHjYfxG/UFCzudbJ8+1KgAETngUXq8RJigPLu0ZHJSQOdgCh2cyJmn2CVQFPJeWngX2VOqJ+6ATi3ZN5+/Zwh9f6PHaOI=

### Context

![today](https://github.com/FlutterFlow/flutterflow-issues/assets/9755885/c91ec575-877b-44f1-afdb-4582dbc5a5b1)
This image shows the components that I deleted yesterday. 

![ff snapshop yesterday](https://github.com/FlutterFlow/flutterflow-issues/assets/9755885/032a3c4d-d489-4349-8c21-1eefc3c3d212)
This image shows the last snapshot from yesterday. You can see that the blocks are not present at that point.

### Visual documentation

2 print screens (snapshot of yesterday) and actual.

### Additional Info

I am not sure if I used FF Desktop yesterday. 

### Environment

```markdown
- FlutterFlow version: 4.1
- Platform: Windows
- Browser name and version: Chrome Versão 123.0.6312.86 (Versão oficial) 64 bits
- Operating system and version affected:
```

**Comment by htsnet** [comment]
New (bad) information: Some files that I changed yesterday and today disappeared too. Until the snapshot at 11:50AM it is OK. The next one, 2:05 PM, some changes was lost and some files returned.

![ff snaphots](https://github.com/FlutterFlow/flutterflow-issues/assets/9755885/d1324a14-2e12-46fc-8265-4d1e63a830e3)

The files in the box I created yesterday. And they were present today, when I published the app.
![snapshot bottomsheet](https://github.com/FlutterFlow/flutterflow-issues/assets/9755885/e9a3b21f-15f9-44ed-ab49-6cb9e5f97e1e)

Now, the files are different:
![without bottomsheet](https://github.com/FlutterFlow/flutterflow-issues/assets/9755885/3d30b347-301b-488b-b50a-975f7cdc9b0a)

To minimize the lost, I restored the version from 11:50AM.
